docker/worker/Dockerfile

```diff
@@ -19,12 +19,18 @@ RUN apt-get update && apt-get upgrade -y
 RUN mkdir /work && mkdir -p /env/docker/worker
 VOLUME /var/lib/docker
 
-# Replicate project structure to make relative editable pipenv dependency work.
+# Replicate project structure to make relative editable poetry dependency work.
 # TODO(ochang): Just copy the entire project (needs a clean checkout).
-COPY docker/worker/Pipfile* /env/docker/worker/
-COPY setup.py Pipfile* README.md /env/
+COPY docker/worker/poetry.lock docker/worker/pyproject.toml /env/docker/worker/
+COPY setup.py poetry.lock pyproject.toml README.md /env/
 COPY osv /env/osv
-RUN cd /env/docker/worker && pipenv install --deploy --system
+
+# Set virtualenv creation to false to install globally
+RUN poetry config virtualenvs.create false
```

Comment:
can we just set `POETRY_VIRTUALENVS_CREATE=false poetry install`  of these?

per https://python-poetry.org/docs/configuration/#virtualenvscreate

Reply:
Good idea!